packages/components/src/Avatar/tests/chromatic/Avatar.stories.tsx

```diff
@@ -0,0 +1,172 @@
+import type { Meta, StoryObj } from "@storybook/react";
+
+import { Inline, Stack } from "../../../layout/index.ts";
+import { AnonymousAvatar, type AnonymousAvatarProps } from "../../src/AnonymousAvatar.tsx";
+import { Avatar } from "../../src/Avatar.tsx";
+import { DeletedAvatar, type DeletedAvatarProps } from "../../src/DeletedAvatar.tsx";
+
+import { Anime, Astronaut, Person } from "./assets/index.ts";
+
+const meta = {
+    title: "Components/Avatar",
+    component: Avatar
+} satisfies Meta<typeof Avatar>;
+
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+type AnonymousStory = StoryObj<AnonymousAvatarProps>;
```

Comment:
you should create 3 different files. AnonymousAvatar, DeletedAvatar and Avatar are 3 different components